start.go

```diff
@@ -9,12 +9,18 @@ import (
 	"path/filepath"
 
 	"github.com/awslabs/goformation/intrinsics"
+	"github.com/fatih/color"
 
 	"github.com/awslabs/aws-sam-local/router"
 	"github.com/awslabs/goformation"
 	"github.com/codegangsta/cli"
 )
 
+// messages color
+var errMsg = color.New(color.FgRed).Add(color.Bold)
```

Comment:
yay. Does this work on Windows?

Reply:
Yes, we are using the [color](https://github.com/fatih/color) library - it also supports Windows.